doc/manual/parallel-computing.rst

```diff
@@ -498,6 +608,71 @@ the backing store on the remote process.
 example of this is provided in ``examples/dictchannel.jl`` which uses a dictionary as its remote store.
 
 
+Channels and RemoteChannels
+---------------------------
+- A :class:`Channel` is local to a process. Worker 2 cannot directly refer to a :class:`Channel`
+  on worker 3 and vice-versa. A :class:`RemoteChannel`, however, can put and take values across
+  workers.
+- A :class:`RemoteChannel` can be thought of as a *handle* to a :class:`Channel`.
+- The process id, ``pid``, associated with a :class:`RemoteChannel` identifies the process where
+  the backing store, i.e., the backing :class:`Channel` exists.
+- Any process with a reference to a :class:`RemoteChannel` can put and take items from the channel.
+  Data is automatically sent to (or retrieved from) the process a :class:`RemoteChannel` is associated with.
+- Serializing  a :class:`Channel` also serializes any data present in the channel. Deserializing
+  it therefore effectively makes a copy of the original object.
+- On the other hand, serializing a :class:`RemoteChannel` only involves the serialization
+  of an identifier that identifies the location and instance of :class:`Channel` referred to
+  by the handle. A deserialized :class:`RemoteChannel` object (on any worker), therefore
+  also points to the same backing store as the original.
+- It is not possible to directly access the backing store of a :class:`RemoteChannel`.
+
+The channels example from above :ref:`man-channels-example` can be modified for
+interprocess communication, as shown below.
+
+We start 4 workers to process a single ``jobs`` remote channel. Job identifiers(``job_id``),
+are written to the channel. Each task in this simulation reads a ``job_id``,
+waits for a random amout of time and writes back a tuple of ``job_id``, time taken and its own ``pid`` to
+the results channel. Finally all the ``results`` are printed out on the master process.
+
+.. doctest::
```

Comment:
again, without any output and a random execution result, does this pass as a doctest?